README.md

```diff
@@ -32,22 +32,66 @@ As Administrator:
 
 ### Docker
 
-**NB** - `cht-conf` in a Docker container should only be used if you're familiar with running containers. You will have to copy or mount your config files into the container in order to be able to use the `cht` call to interact with a remote CHT instance.
+CHT Conf can also be run from within a Docker container. This is useful if you already have Docker installed and do not wish to configure the various dependencies required for developing CHT apps on your local machine. The necessary dependencies are pre-packaged in the Docker image.
 
-Build and then run the `cht-conf` container:
+#### Building the image
 
-	docker build -t cht-conf:v0 .
-	docker run -d --name cht-conf cht-conf:v0
+Run the following commands to create a new project directory and build the Docker image:
 
-You now have a container running in the background called `cht-conf`. You can get a shell on this container with:
+```shell
+mkdir cht-project
+cd cht-project
+curl https://raw.githubusercontent.com/medic/cht-conf/main/Dockerfile > Dockerfile
+docker build -t cht-conf .
+```
 
-	docker exec -it cht-conf /bin/bash
+#### Using the image
 
-And then you can run the utility with `cht`.  
+The resulting Docker image can be used as a [VSCode Development Container](https://code.visualstudio.com/docs/devcontainers/containers) (easiest) or as a standalone Docker utility.
 
-When you are done with the container, you can stop it with:
+##### VSCode Development Container
 
-	docker stop cht-conf
+If you want to develop CHT apps with VSCode, you can use the Docker image as a Development Container. This will allow you to use the `cht-conf` utility and its associated tech stack from within VSCode (without needing to install dependencies like NodeJS on your host system).
+
+[Install VSCode](https://code.visualstudio.com/) if you do not have it already.
+
+Run the following commands from within your project directory to download the `.devcontainer.json` config file, install the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers), and open the project directory in VSCode:
+
+```shell
+curl https://raw.githubusercontent.com/medic/cht-conf/main/devcontainer.config/.devcontainer.json > .devcontainer.json
+code --install-extension ms-vscode-remote.remote-containers
+code -n .
+```
+
+Open the Command Palette in VSCode (_Ctrl+Shift+P_ or _Cmd+Shift+P_) and select `Reopen in Container`. This will open your workspace inside the `cht-conf` container. You can use the `cht` commands by opening a terminal in VSCode (_Ctrl+Shift+\`_ or _Cmd+Shift+\`_).
```

Comment:
```suggestion
Open the Command Palette in VSCode (_Ctrl+Shift+P_ or _Cmd+Shift+P_) and select `Reopen in Container`. This will open your workspace inside the `cht-conf` container. You can use the `cht` commands by opening a terminal in VSCode (_Ctrl+Shift+\`_ or _Cmd+Shift+\`_). If prompted "Do you trust the authors..." choose "Trust Folder & Continue".
```

Reply:
I got a second prompt!